pageserver/src/pgdatadir_mapping.rs

```diff
@@ -90,6 +92,24 @@ pub enum LsnForTimestamp {
     NoData(Lsn),
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub struct LsnRange {
+    pub effective_lsn: Lsn,
+    pub request_lsn: Lsn,
+}
+
+impl LsnRange {
```

Comment:
nit: `Range` in rust usually means everything between start and end, but I can't think of a better name.

Reply:
Neither can I:(